.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
add tabWidth, so that this should change on different IDEs/Code editors.

Reply:
added with value `2` since it's the pattern used at the project